### ISSUE TITLE
Link Sharing: Add People Sharing

### ISSUE THREAD

**Issue by briantho** [photoprism/photoprism#3869]
PLEASE ONLY PROCEED IF YOU ARE ABSOLUTELY SURE THAT THIS IS NOT A TECHNICAL SUPPORT INCIDENT AND/OR A PROBLEM WITH OTHER SOFTWARE YOU ARE USING. PLEASE ALSO CHECK OUR PUBLIC ROADMAP AND TRY TO FIND EXISTING FEATURE REQUESTS FIRST:

- <https://link.photoprism.app/roadmap>
- <https://github.com/photoprism/photoprism/issues>
 
VISIT <https://www.photoprism.app/kb/getting-support> TO LEARN MORE ABOUT OUR SUPPORT OPTIONS. THANK YOU FOR YOUR CAREFUL CONSIDERATION!

===============================================================================

**Describe what problem this solves and why this would be valuable to many users**

Currently link sharing is only possible on Albums / Moments / Calendar / States / Folders according to [this. ](https://docs.photoprism.app/user-guide/share/) It would be nice to share anything in photoprism such as the facial recognition features aka "people". It would also be nice to share videos, and places, but my focus for now would be people. This way I could share all photos of a specific family member with members who i want to share with. For example if i want to share all the photos of my grandma, with one of my cousins, I currently have to give them login access, or create an album for each person, and then share that. Plus, if i were to take new photos of grandma, i would have to add them to the album. Sharing the "people" category simplifies things.

**Describe the solution you'd like**

Prety much explained above, allow sharing for all features of photoprism, such as "People" "Videos" "Places" and "Labels"

**Describe alternatives you've considered**

I'd currently have to create an album for each person i want to share. I would much prefer if we could just share "people" directly. 

**Additional context**

**Comment by brandon1024** [comment]
Interesting idea. I worry a bit that we run the risk of accidentially (unknowingly) sharing pictures we didn't mean to if there's a false match though.

**Comment by lastzero** [comment]
@brandon1024 Exactly, I would expect that to happen quite easily, and that could also be intimate moments.

**Comment by brandon1024** [comment]
Oh I didn't even consider that angle. I don't think implementing a feature like this is a particularly good idea. @briantho  I'd recommend that you instead create a shared album and manually select the images of the person you want to share with them.

**Comment by briantho** [comment]
Ok, i understand your point. But what about the other categories, such as "places" and "videos". Is there a way we could select a general area on the map, and "share" that. I don't think that would be a problem, would it?

**Comment by brandon1024** [comment]
I'd argue you would run into the same problem. For example, you share pictures from a recent trip to Berlin but accidentally include and share pictures from an earlier trip to Berlin that you didn't want to be public.

The only way for the user to be sure that they are sharing exactly the pictures they expect is to limiting sharing to albums, folders and moments, IMO.

**Comment by larionov** [comment]
There is the same functionality in Google Photos and I find it quite useful
